app/services/search/criteria_concocter.rb

```diff
@@ -0,0 +1,72 @@
+# Based on a vacancy, concoct a plausible set of search criteria for a job alert subscription
+class Search::CriteriaConcocter
```

Comment:
I like the word `Concocter` but I had to look it up. Could there be a word that is more common for that?